server/app.js

```diff
@@ -17,6 +17,8 @@ const asyncHandler = require('express-async-handler');
 
 const { DEFAULT_BREW } = require('./brewDefaults.js');
 
+const codeMirrorThemes = require('../themes/codeMirror/editorThemes.json');

```

Comment:
Can we move the loading of this file into the `snippetbar.jsx` perhaps? I don't see it getting used anywhere else, so I'm not sure it needs to be in the global config.

Reply:
`EditorThemes.json` is now copied to the build folder and is loaded in `SnippetBar.jsx` rather than `App.js`.